cpp/log/cert.cc

```diff
@@ -546,12 +588,311 @@ Cert::Status Cert::ExtensionStructure(int extension_nid,
       LOG(WARNING) << "Corrupt extension data";
       LOG_OPENSSL_ERRORS(WARNING);
     }
+
     return FALSE;
   }
 
   return TRUE;
 }
 
+bool IsRedactedHost(const string& hostname) {
```

Comment:
I'm testing the pattern matching separately to avoid the need to create a lot of test certs with bad values.